DigitalLearningSolutions.Data/Services/CourseCompletionService.cs

```diff
@@ -131,6 +131,8 @@ GROUP BY
                          PostLearningPasses.PostLearningPasses,
                          SectionCount.SectionCount;",
                 new { candidateId, customisationId });
+
+            return result?.IncludeCertification == true ? result : null;

```

Comment:
Could you add this to the `WHERE` instead? 🙂 The IncludeCertification shouldn't need to be part of the completion object we return, that would just be confusing because it would seem like it would be possible for it to be false in that case 🙂

Reply:
Yeah that makes sense! Done in f072c1d 🙂